### ISSUE TITLE
Openvino notebook build intermittently fails

### ISSUE THREAD

**Issue by asanzgom** [openvinotoolkit/openvino_notebooks#1398]
**Describe the bug**

Description of problem:

While testing ISVs on RHODS v1.29 RC 2 I got stuck while building an openvino image in the cluster. I keep getting this error in my cluster:

You can remove cached packages by executing 'yum clean packages'.
Error: Transaction test error:
  package nodejs-2:14.18.1-1nodesource.x86_64 does not verify: no digest

error: build error: building at STEP "RUN curl -sL https:/...ec-severity=Moderate": while running runtime: exit status 1

Full logs: [openvino-notebooks-v2022.3-1-undefined.log](https://github.com/secure/attachment/13000145/13000145_openvino-notebooks-v2022.3-1-undefined.log)

I tried in other clusters without being able to reproduce it, however as I faced it, it could be faced by our customers too.

This issue may be out-of-scope for RHODS, but I think we should investigate and report to Intel if something must be changed

Prerequisites (if any, like setup, operators/versions):
Steps to Reproduce
Install OpenVINO operator (either marketplace or certified)
deploy OpenVINO notebook with default settings from OCP Console > Operators > Installed Operators > Openvino toolkit > Notebook (ns: redhat-ods-applications)
go to OCP Console > Builds > Builds
check the build status
Actual results:

build fails

Expected results:

build succeeds

Reproducibility (Always/Intermittent/Only Once):

Intermittent

Build Details:

The bug is still relevant on RHODS1.34 with OCP 4.12 FIPS enabled

**Comment by andrei-kochin** [comment]
Hello @asanzgom,

Thank you for reporting the issue!
Looks like the proper repo to report issues is actually https://github.com/openvinotoolkit/operator but anyway unfortunately I can't access the log you've shared. Could you please double check that it is accessible by the link?

Thank you!

**Comment by avitial** [comment]
@asanzgom do you have an update for us? Seems the log you've shared isn't working.

**Comment by asanzgom** [comment]
Hello @avitial @andrei-kochin sorry I don't know why the link failed: 

[openvino-notebooks-v2022.3-1-undefined.log](https://github.com/openvinotoolkit/openvino_notebooks/files/13417038/openvino-notebooks-v2022.3-1-undefined.log)
  
Error is: 

```
Warning: Push failed, retrying in 5s ...
Registry server Address: 
Registry server User Name: serviceaccount
Registry server Email: serviceaccount@example.org
Registry server Password: <<non-empty>>
error: build error: Failed to push image: writing blob: initiating layer upload to /v2/redhat-ods-applications/openvino-notebooks-v2022.3/blobs/uploads/ in image-registry.openshift-image-registry.svc:5000: received unexpected HTTP status: 500 Internal Server Error
```

**Comment by andrei-kochin** [comment]
@avitial please route this to [openvinotoolkit/operator](https://github.com/openvinotoolkit/operator)
Thank you

**Comment by avitial** [comment]
@andrei-kochin don't have authorization to transfer the issue over to operator, can you help transfer/route the issue?

**Comment by andrei-kochin** [comment]
@avitial  @asanzgom I've cloned the issue to the operator https://github.com/openvinotoolkit/operator/issues/95
